seqr/views/apis/dataset_api_tests.py

```diff
@@ -231,7 +233,8 @@ def test_add_variants_dataset(self, mock_send_email, mock_send_slack, mock_rando
         self.assertListEqual(list(response_json['individualsByGuid'].keys()), ['I000001_na19675'])
         self.assertListEqual(list(response_json['individualsByGuid']['I000001_na19675'].keys()), ['sampleGuids'])
         self.assertSetEqual(set(response_json['individualsByGuid']['I000001_na19675']['sampleGuids']),
-                            {sv_sample_guid, existing_index_sample_guid, new_sample_type_sample_guid, existing_rna_seq_sample_guid})
+                            {sv_sample_guid, existing_index_sample_guid, new_sample_type_sample_guid} |
+                            existing_rna_seq_sample_guids)
```

Comment:
I think we do want to be more careful about testing if some of the new guids we care about are also in `existing_rna_seq_sample_guids`

Reply:
No new guids can be in `existing_rna_seq_sample_guids`, correct? If yes, I can add a test for it.